mp/src/vgui2/vgui_controls/CvarToggleCheckButton.cpp

```diff
@@ -115,6 +115,8 @@ void CvarToggleCheckButton::OnButtonChecked()
     {
         PostActionSignal(new KeyValues("ControlModified"));
     }
+
+    ApplyChanges();
```

Comment:
Should this be in the `HasBeenModified()` branch?

Reply:
True yeah. I made the same mistake in the CvarTextEntry PR #676 . Will update it there as well.